Gabin-ishimwe/myBrand-Portfolio-Project
# #Ud5gqt40  added responsive on pages on my portofolio

### What does this PR do?
in this PR i added responsive on the remaining pages of my portfolio
### Description of Task to be completed?

- fix nav bar
- add hover style on nav bar and icons
- change color layout of my portfolio

### How should this be manually tested?
check responsiveness of the homepage
i deployed manually because the official netlify link is no update with my local repository changes
[portofolio](https://gabinportofolio.netlify.app/index.html)

### Any background context you want to provide?
### What are the relevant pivotal tracker stories?
[uzpSrTH3](https://trello.com/c/uzpSrTH3/29-responsiveness-homepage)

### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/87947641/147251109-105e0d9d-892b-4505-b9f3-80381b5013e8.png)
![image](https://user-images.githubusercontent.com/87947641/147354971-fea29588-3f34-455f-aa0a-7ae538359ebd.png)
![image](https://user-images.githubusercontent.com/87947641/147355039-dadefef7-2a7d-4df1-99b3-a940ec11b6e2.png)


### Questions:
